src/lib/azure/deploymenttable.ts

```diff
@@ -32,6 +34,139 @@ export interface DeploymentEntry {
   pr?: string;
 }
 
+export interface DeploymentRow {
+  status?: string;
+  service?: string;
+  env?: string;
+  author?: string;
+  imageTag?: string;
+  srcToAcrPipelineId?: string;
+  srcToAcrResult?: string;
+  srctoAcrCommitId?: string;
+  AcrToHldPipelineId?: string;
+  AcrToHldResult?: string;
+  AcrToHldCommitId?: string;
+  HldToManifestPipelineId?: string;
+  HldToManifestResult?: string;
+  HldToManifestCommitId?: string;
+  pr?: string;
+  mergedBy?: string;
+  duration?: string;
+  endTime?: string;
+  syncStatus?: string;
+}
+
+/**
+ * Prints deployment table
+ * @param outputFormat output format: json, wide, normal
+ * @param deployments list of deployments to print
+ * @param removeSeparators Whether to remove separators or not
+ */
+export const printDeploymentTable = (
+  outputFormat: OUTPUT_FORMAT,
+  deployments: DeploymentRow[],
+  removeSeparators?: boolean
+): Table => {
+  let header = ["Status", "Service", "Ring"];
+
+  if (outputFormat === OUTPUT_FORMAT.WIDE) {
+    header = header.concat(["Author"]);
+  }
+  header = header.concat(["Image Tag"]);
+  if (!removeSeparators) header = header.concat(["│"]);
+  header = header.concat(["Src to ACR", "Commit", "OK"]);
+  if (!removeSeparators) header = header.concat(["│"]);
+  header = header.concat(["ACR to HLD", "Commit", "OK"]);
+  if (!removeSeparators) header = header.concat(["│"]);
+
+  if (outputFormat === OUTPUT_FORMAT.WIDE) {
+    header = header.concat(["Approval PR", "Merged By"]);
+  }
+  header = header.concat(["HLD to Manifest", "Commit", "OK"]);
+
+  if (!removeSeparators) header = header.concat(["│"]);
+  header = header.concat(["Duration"]);
+  if (outputFormat === OUTPUT_FORMAT.WIDE) {
+    header = header.concat(["End Time"]);
+  }
+  if (outputFormat === OUTPUT_FORMAT.WIDE) {
+    header = header.concat(["Cluster Sync"]);
+  }
+
+  const columnAlignment: Array<"left" | "middle" | "right"> = [];
+  header.forEach((_) => {
+    columnAlignment.push("middle");
```

Comment:
why all are centered?

Reply:
We want to center the columns to make them more readable (see screenshot)

Reply:
i guess this is ok with the UX designer. IMO, columns like Image Tag and Duration should be left and right justified.

Reply:
I missed this. @samiyaakhtar I think we should talk with @gemorris. Either way users have to re-adjust their console window to see everything. 

Doing this by default makes it such that you can't do a `bedrock deployment get > result.tsv`

Reply:
@andrebriggs the command currently in master doesn't output correctly to tsv either, we would need to add support for another output format. A separate issue?

Reply:
My other concern is that by centering the header we have to adjust for the lowest common denominator (aka the longest string value) which highlights our information density issues. Since we seem to be trying to address several items in the epic story https://github.com/microsoft/bedrock/issues/743 at once it might be good to beings folks together to chat about this.